### PR TITLE
Optional quantities

### LINKED ISSUE #12: Make 'quantities' optional
Currently, the wrapper requires every user to have the quantities package installed.
We should check for the availability of the package and adapt functionality if it's not found since the wrapper itself does not require it.

### PR DESCRIPTION
**Pull request by mschmidt87**
This PR makes the quantities optional, i.e., to use the wrapper, one does not need to have the quantities installed. Fixes #12 .

### CODE REVIEW

**Comment by jakobj** [comment]
hmm i think i might have commented at the wrong place. :/

**Comment by mschmidt87** [comment]
Fixed 2 issues.
The AssertEqual issue is a general issue for other tests, so we should adress this in a separate PR.

File: test_wrapper.py [hunk 1]
```diff
@@ -206,15 +213,15 @@ def test_store_and_load_custom_array(self):
...
-        data = {'times': np.array(
-            [1, 2, 3]) * pq.ms, 'positions': np.array([1, 2, 3]) * pq.cm}
+        data = {'times': np.array([1, 2, 3]) * pq.ms, 'positions':
+                np.array([1, 2, 3]) * pq.cm}
         h5w.add_to_h5(fn, data, overwrite_dataset=True)
         # loading the whole data
         res = h5w.load_h5(fn)
         assert(res['times'].dimensionality == data['times'].dimensionality)
-
+
```
Comment (jakobj) [inline test_wrapper.py]: please remove whitespace

**Comment by jakobj** [comment]
except the minor thing above: :+1:

**Pull request merged**